### PR TITLE
chore(*): clean up `package.json` and CircleCI config

### PR DESCRIPTION
**Pull request by gkalpak**
This is a follow-up to #16915, cleaning up `package.json` and `.circleci/config.yml` and making release scripts executable.

### CODE REVIEW

**Comment by petebacondarwin** [comment]
Why two PRs? This and #17033

**Comment by gkalpak** [comment]
I am testing some stuff. (Still drafts - no need to review yet :grin:)

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Configuration file for https://circleci.com/gh/angular/angular
+# Configuration file for https://circleci.com/gh/angular/angular.js
```
Comment (petebacondarwin) [inline .circleci/config.yml:1]: ❤️

**Review by petebacondarwin — APPROVED** [review]
Awesome - thanks for watching my back @gkalpak.

OOC where do you find the SHAs for the docker images?

**Comment by gkalpak** [comment]
🤜🤛

The easiest way to find the SHA (that I know of) is to let the job run and look at the `Spin Up Environment` step of any job that uses the executor you are interested in (e.g. see [here](https://circleci.com/gh/angular/angular.js/829)).

**Review by Splaktar — APPROVED** [review]
LGTM

**Pull request merged**